github/code-scanning.go

```diff
@@ -119,6 +119,16 @@ type AlertListOptions struct {
 	ListOptions
 }
 
+// AnalysisResults specifies the results of a code scanning job.
+//
+// GitHub API docs: https://docs.github.com/en/rest/reference/code-scanning#upload-an-analysis-as-sarif-data
+
```

Comment:
Please delete this blank line so that the auto-generated Go docs get associated correctly with the struct.